### PR TITLE
Add Vega 4 extension

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,5 +25,7 @@ script:
 - jupyter labextension link packages/plotly-extension --no-build
 - jupyter labextension link packages/vega2-extension --no-build
 - jupyter labextension link packages/vega3-extension --no-build
+- jupyter labextension link packages/vega4-extension --no-build
+- jupyter labextension link packages/mathjax3-extension --no-build
```
Comment (domoritz) [inline .travis.yml]: I noticed this was missing so I added it.

**Comment by gnestor** [comment]
I just resolved a merge conflict 👍

**Comment by domoritz** [comment]
Thanks!

**Comment by gnestor** [comment]
I fixed this build issue and merged to master but I can't seem to force push to your branch. Can you `git rebase -i upstream/master`?

**Comment by domoritz** [comment]
Can you just squash and merge with github?

**Comment by gnestor** [comment]
That won't help CI pass, will it?

**Comment by domoritz** [comment]
Nah, that's a bug in a dependency. I don't know why you see that coming up in my PR. It's not related as far as I can see.

**Comment by gnestor** [comment]
It's because your branch is based on the previous master (before the fix). If you rebase to the current master, CI should pass.

**Comment by domoritz** [comment]
Done.

**Comment by ian-r-rose** [comment]
CI is passing, thanks @domoritz!

**Pull request merged**